packages/protocol/src/data/inscription.ts

```diff
@@ -42,6 +42,7 @@ export interface Inscription {
   inscriptionRequest: InscriptionRequest;
   inscriptionSaveRequest: InscriptionSaveRequest;
   inscriptionValidation: InscriptionValidation[];
+  javaType: JavaType[];
```

Comment:
This seems to be coming from another story. May lead to merge conflict with Elio. 
The faster of you two will be happy.